app/views/timeline_events/show.html.erb

```diff
@@ -2,55 +2,108 @@
-        <div class="bg-white border-t p-4 md:p-6">
-          <div class="flex items-center">
-            <div class="shrink-0 w-12 h-12 bg-gray-300 rounded-full overflow-hidden ltr:mr me-3 object-cover">
-              <% if feedback.faculty.user.avatar.attached? %>
-                <%= image_tag feedback.faculty.user.avatar_url(variant: :thumb) %>
-              <% else %>
-                <%= image_tag feedback.faculty.user.initials_avatar %>
-              <% end %>
-            </div>
-            <div>
-              <p class="text-xs leading-tight">Feedback from:</p>
-              <div>
-                <h4 class="font-semibold text-base leading-tight block md:inline-flex self-end"><%= feedback.faculty.name %></h4>
-              </div>
-            </div>
+        <% if presenter.render_markdown?(item) %>
+          <div class="convert-markdown p-1 text-sm" data-json-props="<%= presenter.markdown_prop(item["result"]) %>">
+            <%= item["result"] %>
           </div>
-          <div class="convert-markdown p-3 text-sm" data-json-props="<%= presenter.markdown_prop(feedback.feedback) %>"> <%= feedback.feedback%> </div>
-        </div>
-      <% end %>
+        <% elsif presenter.audio?(item) %>
+          <%= render "audio", file: presenter.audio_file(item) %>
+        <% elsif presenter.files?(item) %>
+          <%= render "files", files: presenter.files(item) %>
+        <% elsif presenter.link?(item) %>
+          <a class="max-w-fc mt-1 me-3 flex border overflow-hidden rounded hover:shadow-md border-blue-400 bg-white text-blue-700 hover:border-blue-600 hover:text-blue-800 focus:ring-2 focus:ring-offset-2 focus:ring-focusColor-500" href="<%= item["result"] %>">
+            <span class="rounded text-sm font-semibold inline-block whitespace-nowrap truncate w-32 md:w-42 h-full px-3 py-2">
+              <%= item["result"] %>
+            </span>
+            <span class="flex w-10 justify-center items-center p-2 bg-blue-700 text-white">
+              <span>
+                <i class="if i-external-link-light if-fw" ></i>
+              </span>
+            </span>
+          </a>
+        <%  else %>
+          <div class="p-3 text-sm"> - <%= item["reslult"].is_a?(Array) ? item["result"].join(" | ") : item["result"] %></div>
+        <% end %>
+      </div>
     </div>
-  </div>
+  <% end %>
+</div>
+<div class="flex flex-col justify-start max-w-2xl mx-auto my-10">
+  <h2 class="font-semibold"><i class="if i-journal-text-regular if-fw mx-2" ></i><%= t(".feedbacks") %></h2>
+  <% unless @submission.startup_feedback.present? %>
+    <p class="pl-5">No feedbacks present.</p>
+  <% end %>
+  <% @submission.startup_feedback.includes(faculty: [user: [avatar_attachment: :blob]]).each do |feedback| %>
+    <div class="flex bg-gray-50 p-4 rounded-md mt-2">
+      <div class="shrink-0 w-8 h-8 bg-gray-300 rounded-full overflow-hidden ltr:mr me-3 object-cover">
```

Comment:
You don't need `ltr:mr`. If you are using logical inline CSS `me-*`

BTW, `mr` is not a valid class in Tailwind.